flips/20220811-fungible-tokens-metadata.md

```diff
@@ -0,0 +1,210 @@
+# Fungible Tokens Metadata
+
+| Status        | Draft       |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #1087**    | [1087](https://github.com/onflow/flow/pull/1087)|
+| **Author** | Álvaro Lillo Igualada (alvaro.lillo@dapperlabs.com) |
+| **Sponsor**   | Josh Hannan (joshua.hannan@dapperlabs.com)             |
+| **Updated**   | 2022-08-16                                           | 
+
+## Objective
+
+This proposal, in a similar fashion as the NFT Metadata, will make possible to create generic solutions that will interoperate through views rather than through hard-coded types. This will allow better interoperability between fungible tokens and external applications. 
+
+## Motivation
+
+As the Flow ecosystem continues growing the need of assuring contract and external apps interoperability increases. The development of the NFT Metadata standards brought awareness of how important is the way any token on Flow is able to communicate its features to other dapps, and of the advantages of doing so in a standardized way.
+
+## User Benefit
+
+The creation of a Metadata standard for fungible tokens will bring two major user benefits, discoverability and interoperability. It will make way easier to any FT to be presented on any app that would want to feature several FT and also will ease how those FT can be used by an user programmatically.
```

Comment:
```suggestion
The creation of a Metadata standard for fungible tokens will bring two major user benefits, discoverability and interoperability. It will make it way easier for any FT to be presented on any app that would want to feature several FT and also will ease how those FT can be used by a user programmatically.
```